sos/report/plugins/maas.py

```diff
@@ -11,44 +11,39 @@
 from sos.report.plugins import Plugin, UbuntuPlugin, PluginOpt
 
 
-class Maas(Plugin, UbuntuPlugin):
+class MAAS(Plugin, UbuntuPlugin):
 
-    short_desc = 'Ubuntu Metal-As-A-Service'
+    short_desc = 'MAAS | Metal as a Service'
 
     plugin_name = 'maas'
+    plugin_timeout = 1800
     profiles = ('sysmgmt',)
-    packages = ('maas', 'maas-common')
```

Comment:
I think its worthwhile keeping this here, I am hoping with the work we've been doing with the `--all-logs` issue, it will parse this and will improve the snap package detection in #3684

Reply:
Returned `packages` back with a more explicit list.